packages/oui-guide-dropdown/src/guide.html

```diff
@@ -0,0 +1,26 @@
+<oui-dropdown arrow align="end" class="oui-guide">
+  <button class="oui-guide__button" oui-dropdown-trigger aria-label="guide">
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 23 17.15" width="22px">

```

Comment:
It would be preferable to import guide icon into ui-kit icon to use it :)

Reply:
I tried to. But the svg is not getting converted to icon using icon font generator. Tried converting to simple path too but not able to get the desired result. Have requested CX team to get the icon.